integrations/cesium/CesiumEuclideonIntegration.html

```diff
@@ -53,7 +53,15 @@ <h1>Loading...</h1>
       var udSDKReady = 0;
 
       var viewer = new Cesium.Viewer("cesiumContainer", {
-        scene3DOnly: true
+        scene3DOnly: true,
+        imageryProvider: new Cesium.UrlTemplateImageryProvider({
+          url: 'https://slippy.vault.euclideon.com/aerial/{z}/{x}/{y}.png',
+          credit: new Cesium.Credit(`© ${new Date().getFullYear()} TomTom, Microsoft`, true),
```

Comment:
credit is the data source, shouldn't it be Euclideon?

Reply:
@asunawu  Credit should be to the original creator I think...